cmd/cape/cmd/token.go

```diff
@@ -67,12 +81,24 @@ func token(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	_, err = cmd.OutOrStdout().Write([]byte(tokenString + "\n"))
-	if err != nil {
-		return err
+	if o != "json" {
+		_, err = cmd.OutOrStdout().Write([]byte(tokenString + "\n"))
+		if err != nil {
+			return err
+		}
 	}
 
-	return nil
+	output := struct {
+		ID       string `json:"function_id"`
+		Token    string `json:"function_token"`
```

Comment:
hmm I think I am missing some conversion here for the function token. For the stdout we return `[]byte(tokenString + "\n")` when in this struct we just return a string. The format end up not being the same if the output is json or not.